# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/cors/redirect-userinfo.htm.ini
# Commit: 9ad299b5161b
# Full Hash: 9ad299b5161b0cedf852d56365310de499f94043
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2018-11-14 22:12:17
# Regressor Bug: 1495313
# File Overlap Count: 1
# Description:
#   Bug 1495313 - Allow empty username in URLs r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D11254
# ==============================================================================

diff -r 6293789f6e9c -r 9ad299b5161b testing/web-platform/meta/cors/redirect-userinfo.htm.ini
--- a/testing/web-platform/meta/cors/redirect-userinfo.htm.ini	Wed Nov 14 14:27:58 2018 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,4 +0,0 @@
-[redirect-userinfo.htm]
-  [Allow redirect without userinfo (:@ is trimmed during URL parsing)]
-    expected: FAIL
-